packages/components/src/unit-control/types.ts

```diff
@@ -0,0 +1,38 @@
+export interface WPUnitControlUnit {
```

Comment:
We usually prefer `type` over `interface`

(here's an [example why](https://fettblog.eu/tidy-typescript-prefer-type-aliases/))